NEXTGEN_DESIGN_PROPOSAL.md

```diff
@@ -0,0 +1,62 @@
+# Design Proposal for Project "Dash"
+
+The openshift-applier has done well in its current shape as an ansible playbook paired with an inventory, driving what content to seed an openshift-cluster with. However, it also has some flaws that now can be better addressed with newer technologies and approaches that are becoming mainstream. This document captures the proposed design of the next iteration of this project, with new branding and a new contract.
+
+The working name for the project is "Dash" (because when you remove the words `openshift` and `applier`, you're left with `-`).
+
+## Design Philosophy
+
+TODO
+
+## Desired Feature List
+
+- Dash should take the form of a CLI tool
```

Comment:
Do we want to mention anything about it being a library that can be consumed directly (in addition to the CLI option)?

Reply:
@oybed done.